### PR TITLE
Doc tokens for #1792 #1766 #1103

### PR DESCRIPTION
**Pull request by fgalan**
Implement documentation items for #1792, #1766, #1103 

@kzangeli 
@crbrox 
@jmcanterafonseca (in the case you have the time and opportunity)

### CODE REVIEW

File: doc/apiary/v2/fiware-ngsiv2-reference.apib [hunk 1]
```diff
@@ -659,7 +660,11 @@ Response code:
     + offset: 20 (optional, number) - Establishes the offset from which entities will be returned
     
     + attrs: seatNumber (optional, string) - Comma-separated list of attribute names which data will be included in the response.
-    If this parameter is not included, all the attributes are retrieved.
+    The attributes are retreived in the same order as specified by this parameter. If this parameter is not included, all the attributes
+    are retrieved, in arbitrary order.
```
Comment (kzangeli) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: Remove comma.

And a question ...  Why use  *retrieved* (misspelled btw) and not *returned* ?

**Comment by kzangeli** [comment]
I believe every **in the case** in this doc should be replaced by **in case**.
These two constructions have different meanings ...

File: doc/apiary/v2/fiware-ngsiv2-reference.apib (hunk 1 above)
Comment (kzangeli) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: For the entire doc:
in the same order as specified by  => in the order specified by

Comment (fgalan) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: > Why use retrieved (misspelled btw) and not returned ?

I think first versions of the .apib used "retrieved" and we have continued using it.

**Comment by fgalan** [comment]
> I believe every in the case in this doc should be replaced by in case.
> These two constructions have different meanings ...

Let's try to keep focused in the changes in this PR. Doing overall changes right now could have effect in other existing PRs on the .apib (in particular the one about notifications).

Of course, this kind of things needs to be fixed... but in an overall "proof read" process once the .apib get finisehd.

File: doc/apiary/v2/fiware-ngsiv2-reference.apib (hunk 1 above)
Comment (fgalan) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: Fixed in 4979cbe

**Comment by kzangeli** [comment]
Sure, LGTM

**Pull request merged**